ql/src/test/queries/clientpositive/cte_mat_10.q

```diff
@@ -0,0 +1,23 @@
+set hive.optimize.cte.materialize.threshold=2;
+set hive.optimize.cte.materialize.full.aggregate.only=false;
+set hive.explain.user=true;
```

Comment:
Why did user.explain set to true? The plan has more details when it is false.

Reply:
I can no longer find the reason since this case was written 3 years ago. Anyway, I removed it and reran all test cases.
- [Stopped setting hive.explain.user, added EXPLAIN, and reran 3 test cases](https://github.com/apache/hive/pull/4363/commits/368f66e5425524aa3fbd0b32b37227ecab558d73)
- [Merged master because I found some of new test cases are affected by other PRs](https://github.com/apache/hive/pull/4363/commits/3a37c9729f8728c32bc33fce18b7c66dddc56ff3)
- [Reran the tests again](https://github.com/apache/hive/pull/4363/commits/de051af22caebb7e533927371ac39d111a130921)

I am waiting for CI to finish